satpy/composites/viirs.py

```diff
@@ -72,9 +71,14 @@ def __call__(self, datasets, **info):
 
         dnb_data = datasets[0]
         sza_data = datasets[1]
-        delayed = dask.delayed(self._run_dnb_normalization)(dnb_data.data, sza_data.data)
+        output_data = da.map_blocks(
+            self._run_dnb_normalization,
+            dnb_data.data.rechunk(dnb_data.shape),
+            sza_data.data.rechunk(sza_data.shape),
```

Comment:
just one questio that just popped up: are these two guaranteed to have the same shape? otherwise `unify_chunks` might create multiple chunks for one of the arrays at least...